.gitignore

```diff
@@ -10,6 +10,7 @@
 *.syso
 .idea
 *.DS_Store
+script.js
```

Comment:
```suggestion
# Excluding as it is the file generated by the k6 init command
script.js
```
or something similar so if someone isn't aware of the init command then they can understand why it is there